src/patternfly/components/Table/table-scrollable.scss

```diff
@@ -43,12 +43,14 @@
 
 .pf-c-scroll-outer-wrapper {
   --pf-c-scroll-outer-wrapper--MinHeight: #{pf-size-prem(400px)};
+  --pf-c-scroll-outer-wrapper--MaxHeight: 100%;
 
   display: flex;
   flex-direction: column;
   max-width: 100%;
+  height: 100%;
```

Comment:
did you mean to add this? I don't see reference to it and looks like it could change an existing layout if the `min-height` was unset to allow a table to grow to the `max-height`, then start scrolling. 

min-height auto

<img width="1840" alt="Screen Shot 2022-08-18 at 5 56 29 PM" src="https://user-images.githubusercontent.com/35148959/185509367-78485a2f-0893-4a63-aea8-c47694de77e0.png">

^ w/ height: 100%

<img width="1840" alt="Screen Shot 2022-08-18 at 5 56 33 PM" src="https://user-images.githubusercontent.com/35148959/185509357-d8eee83f-a8c1-47a0-af9e-49d63adb914a.png">

From what I tested, `height: 100%` doesn't seem necessary (at least in the codesandbox @kmcfaul created), but I don't know all of the use cases.

Reply:
> did you mean to add this?

@mcoker oops, nope